### DISCUSSION TITLE
[Item] （已适配7，解决globalCache报错）「阅读成果量化」精简高效中英文献精读分析模板，自用

### DISCUSSION THREAD

**Discussion by infinitesum** [windingwind/zotero-better-notes#903] [Note Templates]
### Description

基于
[[item] (已适配7) 自用读中文文献、读英文文献笔记模板两套，简约风格](https://github.com/windingwind/zotero-better-notes/discussions/729)修改而来

zotero 7 版本已修复   `Zotero.ZoteroStyle.data.ztoolkit.ItemTree.fieldHooks.globalCache is undefined` 报错

### 常见报错及解决

其实大部分都已经说过了但是问的人太多还是更新一下，如果你没有成功的生成笔记，请自查是否属于下列情况：

- 请首先升级到最新版插件，包括相应的翻译插件；
- 文献没有附件。会报错 `Template [item]qweiqwe Error: Error: getAttachments() cannot be called on attachment items`。没有附件想要生成模板笔记请在代码中删除 PDF 附件部分
```
<tr>
<td><b>原文PDF链接: </b>
<!-- 前提是有pdf附件，否则也找不到 -->
<a href=zotero://open-pdf/0_${Zotero.Items.get(topItem.getAttachments()).filter((i) => i.isPDFAttachment())[0].key}>${Zotero.Items.get(topItem.getAttachments()).filter((i)=> i.isPDFAttachment())[0].getFilename()} 
    </a>
</td>
</tr>
```

- 选中了孤立的 PDF 条目。要选中文献条目而不是 PDF 条目，否则也会报错；
- 模版未正确命名。模版的名称前面要加 [item]。

### Screenshots

中文文献：
![CleanShot 2024-03-03 at 20 20 53@2x](https://github.com/windingwind/zotero-better-notes/assets/148856002/603a8360-79b3-401b-9a5c-f93e50ef0dfa)
![CleanShot 2024-03-03 at 20 21 06@2x](https://github.com/windingwind/zotero-better-notes/assets/148856002/6886aeea-4a6c-4892-9b11-136dc28b80b4)

英文文献：
![CleanShot 2024-03-03 at 20 16 02@2x](https://github.com/windingwind/zotero-better-notes/assets/148856002/fe037ec9-0996-45cd-8205-035712e69151)
![CleanShot 2024-03-03 at 20 17 03@2x](https://github.com/windingwind/zotero-better-notes/assets/148856002/fd2a7bdd-0ca1-4f01-89ff-9ca006459b67)

### Test on Zotero version

6.0.33

### Test on Better Notes version

1.0.4

### Template Type

Item

### Zotero 6代码

```yaml
<html>
<hr/>
<table border="1">

<h2 style="color:  #E65100; background-color:  #FFF8E1;">(${topItem.getField("date")}) ${topItem.getField("title")}</h2>

<tr>
<td><b>期刊: <b style="color:#FF0000">${topItem.getField('publicationTitle')}</b></b>（发表日期: <b>${topItem.getField("date")}</b>）<br><b>作者:</b> ${topItem.getCreators().map((v)=>v.firstName+" "+v.lastName).join("; ")}</td>
</tr>

<tr>
<td><b>摘要: </b><i>${topItem.getField('abstractNote')}</i></td>
</tr>

<tr>
<td><b>摘要翻译: </b>
<!-- 要提前安装Zotero PDF Translate插件，否则摘要翻译找不到 -->
<i>${topItem.getField('abstractTranslation')}</i></td>
</tr>


<tr>
<td><b>期刊分区: </b>
<!-- 引用了zotero style插件的标签，请提前安装，否则找不到分区 -->
${(() => {let space = " ㅤㅤ ㅤㅤ"
                return Array.prototype.map.call(Zotero.ZoteroStyle.data.ztoolkit.ItemTree.globalCache.renderCellHooks.PublicationTags(
      0,
      Zotero.ZoteroStyle.data.ztoolkit.ItemTree.fieldHooks.globalCache.getFieldHooks.PublicationTags(
      "", true, true,
      topItem, undefined)
      ).childNodes,
      e => {
      e.innerText = space + space + e.innerText + space + space;
      return e.outerHTML
      }).join(space)
      })()
      }
</td>
</tr>

<tr>
<td><b>原文PDF链接: </b>
<!-- 前提是有pdf附件，否则也找不到 -->
<a href=zotero://open-pdf/0_${Zotero.Items.get(topItem.getAttachments()).filter((i) => i.isPDFAttachment())[0].key}>${Zotero.Items.get(topItem.getAttachments()).filter((i)=> i.isPDFAttachment())[0].getFilename()} 
    </a>
</td>
</tr>

<tr>
<td><b>笔记创建日期: </b>${new Date().toLocaleString()}</td>
</tr>

</table>
<!-- Tips不想要可以自行删掉 -->

<section>
    <h2 id="background-purpose-conclusion">1️⃣ 文章基本框架</h2>
    
    <h3>背景</h3>
<blockquote>
        概要说明研究所处领域的现状及其重要性。
    </blockquote>
    <h3>目的</h3>
<blockquote>
        明确列出文章旨在解决的科学问题和研究目标。
    </blockquote>
    <h3>结论</h3>
<blockquote>
简明扼要地归纳出文章的主要发现与结论。
</blockquote>

</section>
<br>

<section>
    <h2 id="results-discussion">2️⃣ 结果与讨论</h2>
    <blockquote>
        对于文章中出现的关键图表及其说明、关键机理解释以及文章的重点内容，进行详细记录和解读。
    </blockquote>
    <h3>数据以及数据来源</h3>
    <h3>方法</h3>
    <h3>结论</h3>
</section>
<br>

<section>
    <h2 id="thoughts">3️⃣ 文章亮点思考</h2>
    <blockquote>
        <p>1、本文有什么优缺点？你是否对某些内容产生了疑问？<br>
        2、你是否认为某些研究方式可以改进，如何改进？<br>
        3、这篇文章好在哪里，只有当自己理解这篇文章好在哪里，为什么能发在顶刊上，那么你也就基本理解了全篇文章的基本内容。</p>
    </blockquote>

</section>

<br>
<section>
    <h2 id="learning-by-example-125">4️⃣ 借鉴学习（125）</h2>
    <blockquote>
        <p>“125”原则</p>
    </blockquote>
    <h3>1个思路</h3>
    <blockquote>
        <p>该文章可能为自己未来研究带来的新思路，不论是实验方法，还是研究思路上。</p>
    </blockquote>
    <h3>2个绘图</h3>
    <blockquote>
        <p>选取并分析文章中设计精良的图表，学习其形式和色彩搭配。</p>
    </blockquote>
    <h3>5个句式</h3>
    <blockquote>
        <p>提炼并记录文章中的五个优秀句式，并尝试在未来的写作中模仿使用。</p>
    </blockquote>
</section>

</html>
```

### Zotero 7 代码

```yaml
# This template is specifically for importing/sharing, using better 
# notes 'import from clipboard': copy the content and
# goto Zotero menu bar, click Edit->New Template from Clipboard.  
# Do not copy-paste this to better notes template editor directly.
name: "[item]note"
content: |-
  <html>
  <hr/>
  <table border="1">
  
  <h2 style="color:  #E65100; background-color:  #FFF8E1;">(${topItem.getField("date")}) ${topItem.getField("title")}</h2>
  
  <tr>
  <td><b>期刊: <b style="color:#FF0000">${topItem.getField('publicationTitle')}</b></b>（发表日期: <b>${topItem.getField("date")}</b>）<br><b>作者:</b> ${topItem.getCreators().map((v)=>v.firstName+" "+v.lastName).join("; ")}</td>
  </tr>
  
  <tr>
  <td><b>摘要: </b><i>${topItem.getField('abstractNote')}</i></td>
  </tr>


  <tr>
  <td><b>摘要翻译: </b>
  <!-- 要提前安装Zotero PDF Translate插件，否则摘要翻译找不到 -->
  <i>${topItem.getField('abstractTranslation')}</i></td>
  </tr>

  
  <tr>
  <td><b>期刊分区: </b>
  <!-- 引用了zotero style插件的标签 -->
  
  ${{
  let space = " ㅤㅤ ㅤㅤ"
  return Array.prototype.map.call(
  	Zotero.ZoteroStyle.api.renderCell(topItem, "publicationTags").childNodes,
  	e => {
  		e.innerText =  space + e.innerText + space;
  		return e.outerHTML
  	}
  	).join(space)
  }}$
  
  
  </td>
  </tr>
  
  <tr>
  <td><b>原文PDF链接: </b>
  <a href=zotero://open-pdf/0_${Zotero.Items.get(topItem.getAttachments()).filter((i) => i.isPDFAttachment())[0].key}>
                  ${Zotero.Items.get(topItem.getAttachments()).filter((i)=> i.isPDFAttachment())[0].getFilename()}
      </a>
  </td>
  </tr>
  
  <tr>
  <td><b>笔记创建日期: </b>${new Date().toLocaleString()}</td>
  </tr>
  
  </table>
  <section>
      <h2 id="background-purpose-conclusion">1️⃣ 文章基本框架</h2>
      
      <h3>背景</h3>
  <blockquote>
          概要说明研究所处领域的现状及其重要性。
      </blockquote>
      <h3>目的</h3>
  <blockquote>
          明确列出文章旨在解决的科学问题和研究目标。
      </blockquote>
      <h3>结论</h3>
  <blockquote>
  简明扼要地归纳出文章的主要发现与结论。
  </blockquote>
  
  </section>
  <br>
  
  <section>
      <h2 id="results-discussion">2️⃣ 结果与讨论</h2>
      <blockquote>
          对于文章中出现的关键图表及其说明、关键机理解释以及文章的重点内容，进行详细记录和解读。
      </blockquote>
      <h3>数据以及数据来源</h3>
      <h3>方法</h3>
      <h3>结论</h3>
  </section>
  <br>
  
  <section>
      <h2 id="thoughts">3️⃣ 文章亮点思考</h2>
      <blockquote>
          <p>1、本文有什么优缺点？你是否对某些内容产生了疑问？<br>
          2、你是否认为某些研究方式可以改进，如何改进？<br>
          3、这篇文章好在哪里，只有当自己理解这篇文章好在哪里，为什么能发在顶刊上，那么你也就基本理解了全篇文章的基本内容。</p>
      </blockquote>
  
  </section>
  
  <br>
  <section>
      <h2 id="learning-by-example-125">4️⃣ 借鉴学习（125）</h2>
      <blockquote>
          <p>“125”原则</p>
      </blockquote>
      <h3>1个思路</h3>
      <blockquote>
          <p>该文章可能为自己未来研究带来的新思路，不论是实验方法，还是研究思路上。</p>
      </blockquote>
      <h3>2个绘图</h3>
      <blockquote>
          <p>选取并分析文章中设计精良的图表，学习其形式和色彩搭配。</p>
      </blockquote>
      <h3>5个句式</h3>
      <blockquote>
          <p>提炼并记录文章中的五个优秀句式，并尝试在未来的写作中模仿使用。</p>
      </blockquote>
  </section>
  
  
  </html>
```

**Comment by Stan-TT** [comment]
请问我导入模板后，显示Template [item]qweiqwe Error: Error: getAttachments() cannot be called on attachment items，这个错误是为什么呢？求解答（抱拳）

**Reply by Eddie0521** [reply]
把里面pdf附件那部分删掉就可以了

**Reply by XURTong** [reply]
> 把里面pdf附件那部分删掉就可以了

请问具体是哪部分，没有找到

**Reply by hsjyooo** [reply]
![image](https://github.com/user-attachments/assets/38e3facd-2959-4bae-bfb7-49790f10518e)
把这部分删掉就可以了

**Reply by Jingyukong** [reply]
感谢提问和解答，遇到同样的问题，已经解决啦

**Comment by Chimeri3721** [comment]
你好，我在使用模板时，英文文献不会生成摘要翻译，这是什么原因导致的呢？我的zotero版本是beta65，BN版本是1.1.4beta56，translate是1.1.0.beta33，都是目前的最新版本。

**Reply by infinitesum** [reply]
6的是可以正常用的，但是我在zotero 7上测试的时候，摘要翻译显示不到这里
![CleanShot 2024-03-10 at 16 50 05@2x](https://github.com/windingwind/zotero-better-notes/assets/148856002/8ac0265c-052d-48a6-870c-9f4d6d618f3b)
暂时不知道是什么原因，可能是翻译插件的bug，所以在7的模板里去掉了这一部分，后面我再看看吧。

**Reply by windingwind** [reply]
更新翻译插件即可

**Reply by Chimeri3721** [reply]
更新翻译插件之后确实可以正常显示了，作者大大@infinitesum可以辛苦把那个功能加上了，感谢@windingwind

**Reply by infinitesum** [reply]
已更新

**Reply by Chimeri3721** [reply]
多谢多谢👋

**Reply by wyy-yiyang** [reply]
为什么我更新翻译插件也不行......

**Reply by wyy-yiyang** [reply]
更新后可以了

**Reply by zhao-zhibo** [reply]
@wyy-yiyang 你好，我也是更新了 最新的zotero6的翻译插件 可是还是不行

**Reply by wyy-yiyang** [reply]
我用的zotero7，翻译插件注意使用的是6的版本还是7的版本。也可以使用插件市场

**Reply by XiaotongSun980917** [reply]
我用的也是zotero7，刚安装的插件，为啥也不行了，没有翻译， 插件市场是啥呀

**Comment by DFBlowing** [comment]
感谢作者大大，终于可以用了，嘻嘻😁

**Comment by haojieya** [comment]
#你好，请问我保存完笔记模板之后报错“Template [item]intensive_reading Error: SyntaxError: missing : after property id”

**Reply by limmmer** [reply]
我也是这样的

**Reply by limmmer** [reply]
你解决了么？

**Reply by Li-hr** [reply]
我也是这样，咋回事

**Reply by windingwind** [reply]
升级插件

**Reply by Anthonycccc** [reply]
工具→插件市场→Better Notes，更新一下可以解决

**Comment by mingxi-nuli** [comment]
报错：  Template 中文文献 Error: ReferenceError: topItem is not defined

**Reply by reoLantern** [reply]
遇到了同样的问题

**Reply by Oliver-zpf** [reply]
在模版的名称前面加 [item]
<img width="137" alt="image" src="https://github.com/windingwind/zotero-better-notes/assets/147411209/fd2bbae2-7ae0-4060-8251-10a51fc955aa">
就像最后一个一样

**Comment by wyy-yiyang** [comment]
解决了原模板报错，棒棒棒 Error: TypeError: Zotero.ZoteroStyle.data.ztoolkit.ItemTree.fieldHooks.globalCache is undefined

**Reply by twoeggs1109** [reply]
请问如何解决的呀

**Reply by ceciliayan** [reply]
我也是这个问题

**Comment by 24always** [comment]
您好，我的导入模板后显示SyntaxError: missing : after property id 。求作者大大帮忙！

**Reply by YiT8488** [reply]
我也是 请问你是MacBook吗

**Comment by yzm19871218** [comment]
感谢可以使用 作者大大厉害

**Comment by wyy-yiyang** [comment]
> @wyy-yiyang 你好，我也是更新了 最新的zotero6的翻译插件 可是还是不行

我用的zotero7，翻译插件注意使用的是6的版本还是7的版本。也可以使用插件市场

**Comment by djiuhong** [comment]
您好！我使用的是Zotero7.0.0-beta.76+91054acfe，Better Notes for Zotero1.1.4-25，请问为什么我复制模版后，在编辑里点击New Template from Clipboard没有任何反馈，也没有新的模版被导入（之前这个功能一直是可以使用的

**Reply by djiuhong** [reply]
更新BN到[1.1.4-beta.72]后,同样的操作会显示“The copied template is invalid”

**Reply by bitorangej** [reply]
+1

**Reply by infinitesum** [reply]
直接粘贴就行了，不是模板的问题哈

**Comment by haojieya** [comment]
我的是Windows


---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年05月06日 17:05 |
| 收件人 | windingwind/zotero-better-notes ***@***.***> |
| 抄送人 | haojieya ***@***.***>,
Comment ***@***.***> |
| 主题 | Re: [windingwind/zotero-better-notes] [Item] （已适配7，解决globalCache报错）「阅读成果量化」精简高效中英文献精读分析模板，自用 (Discussion #903) |

我也是 请问你是MacBook吗

**Comment by xiejecen** [comment]
![image](https://github.com/windingwind/zotero-better-notes/assets/72964137/6828ca01-60a5-4444-b355-eab451ba5819)
我是最新版的导入模板无效

**Reply by bitorangej** [reply]
加一

**Reply by xiejecen** [reply]
怎么解决呀

**Reply by infinitesum** [reply]
直接粘贴就行了，不是模板的问题哈

**Reply by xiejecen** [reply]
感谢作者大大，还有一个问题就是摘要不翻译怎么解决呀

**Reply by infinitesum** [reply]
更新翻译插件

**Comment by Eddie0521** [comment]
又排查了一下搞清楚了，选择的时候不能只选孤立的pdf文件，选择条目即可



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年05月30日 08:22 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [windingwind/zotero-better-notes] [Item] （已适配7，解决globalCache报错）「阅读成果量化」精简高效中英文献精读分析模板，自用 (Discussion #903) |

直接粘贴就行了，不是模板的问题哈

**Reply by ArwenYu** [reply]
试了一下，Template Preview Error: Error: getAttachments() cannot be called on attachment items问题可以用这个方法解决，谢谢楼主。

**Comment by npj06** [comment]
<img width="510" alt="截屏2024-06-03 11 05 40" src="https://github.com/windingwind/zotero-better-notes/assets/114918453/62f02d4a-3e0c-4e51-9d77-e26e895a95dc">
请问作者大大，如何解决，感谢

**Comment by Aliniya128** [comment]
请问摘要分为几个部分时获取失败怎么办呀，像这种
![Uploading dd3fbb9c-219a-4dc5-b4a0-fc76c584e25f.png…]()

**Comment by Aliniya128** [comment]
![dd3fbb9c-219a-4dc5-b4a0-fc76c584e25f](https://github.com/user-attachments/assets/4184f20c-c306-4296-9607-64b7d648ff3d)

**Comment by meilin17** [comment]
插入模板后一直显示
Template [item]文献阅读 Error: TypeError: Zotero.Items.get(...).filter(...)[0] is undefined

**Reply by PtheFlg** [reply]
您好，解决了嘛，这个问题

**Reply by wangfly1993** [reply]
这个怎么解决的？

**Reply by dawsonmax** [reply]
解决了吗？

**Reply by Nova-3-sy** [reply]
同样的问题，请问如何解决呀555

**Reply by SSF-max** [reply]
同样的问题，请问怎么解决的呢？

**Reply by ly7900** [reply]
删掉上面作者发的那一段就好了
就这段
<tr>
<td><b>原文PDF链接: </b>

<a href=zotero://open-pdf/0_${Zotero.Items.get(topItem.getAttachments()).filter((i) => i.isPDFAttachment())[0].key}>${Zotero.Items.get(topItem.getAttachments()).filter((i)=> i.isPDFAttachment())[0].getFilename()} 
    </a>
</td>
</tr>

**Comment by Aurora9211** [comment]
![WeChatf190c3ff900a8a66862f951e3663a572](https://github.com/user-attachments/assets/138b2bce-7e8b-4ab5-986e-a3a04c4efcee)
这个问题要怎么解决呢

**Reply by Aurora9211** [reply]
我从剪贴板粘贴显示也是这样

**Reply by windingwind** [reply]
确保复制的是正确版本的代码

**Reply by LaixSS** [reply]
粘贴完保存的时候问 似乎您正在尝试直接保存一个笔记模板分享代码。您想要将其导入为模板吗？，选择取消就行了

**Reply by windingwind** [reply]
通常来说，如果提示你是否要导入为模板，应选择是。

**Reply by LaixSS** [reply]
> 通常来说，如果提示你是否要导入为模板，应选择是。

其他模板确实选择是，但是这个模板选择是就会出问题，取消了才行，不知道为啥

**Reply by windingwind** [reply]
你确定复制对了内容吗？复制上面的zotero7对应模板

**Reply by LaixSS** [reply]
> 你确定复制对了内容吗？复制上面的zotero7对应模板

我确认复制的是正确的，就是有问题

**Reply by windingwind** [reply]
检查是模板有问题，已更正。原作者在修改模板内容的时候，未在模板编辑器中修改后导出，而是直接修改了导出的代码，其中缩进有误，因此无法识别。

**Reply by LaixSS** [reply]
> 检查是模板有问题，已更正。原作者在修改模板内容的时候，未在模板编辑器中修改后导出，而是直接修改了导出的代码，其中缩进有误，因此无法识别。

好的，谢谢大佬

**Comment by yyk0128** [comment]
请问大佬这个怎么解决呀
Template Preview Error: TypeError: Zotero.ZoteroStyle.data.ztoolkit.ItemTree.globalCache is undefined

**Comment by ghost** [comment]
请问为什么我生成的条目笔记，没有标题，求大佬解答
![Clip_2024-09-25_16-45-48](https://github.com/user-attachments/assets/8cf6b955-cd1e-4d5c-8739-f17c9127033e)

**Reply by hbczgs** [reply]
我把红框内删掉；把h2 改为h1   生成条目笔记就有标题了。
![image](https://github.com/user-attachments/assets/a4136de4-a158-45a5-9494-f7a04a813995)

**Reply by ghost** [reply]
非常感谢，确实是这样的！

**Comment by JaneWVv** [comment]
请问zotero7的translate升级到2.0.1还是不翻译摘要是什么原因呢？非常感谢大佬的回复！; >

**Comment by yuijkl890** [comment]
作者你好，我导出笔记的PDF版后，包括（期刊名、摘要、摘要翻译）的这个表格打印不全，但是后面的笔记可以完整导出到PDF，请问怎么操作可以导出完整的笔记PDF

**Comment by mailejikuai** [comment]
请问生成笔记之后，我打开pdf看文章为什么找不到笔记呢

**Comment by Songsongszu** [comment]
我的为什么报语法错误呀，Preview not available for template

**Reply by Songsongszu** [reply]
![image](https://github.com/user-attachments/assets/55926f38-4232-4a4b-bb5b-adfcedec8a6a)

**Reply by Robs87** [reply]
同问

**Reply by lizhibale** [reply]
同问

**Comment by ludan926** [comment]
找不见Zotero pdf translate， 安装了translate for Zotero  ，无法显示摘要翻译

**Reply by mortyandtimo** [reply]
一样的问题

**Comment by huxxu** [comment]
请问作者大大，能根据标签自动生成笔记嘛。如果改的话怎么办呀

**Comment by Rokiyaaaa** [comment]
Error: TypeError: Zotero.ZoteroStyle.data.ztoolkit.ItemTree is undefined（style插件是最新的呀为什么还报错啊）

**Comment by YajingXu** [comment]
![QQ_1744254095461](https://github.com/user-attachments/assets/08d6869f-4bd4-49a4-b5fd-8101be46cc5b)
请问大家，我上传之后笔记内容是空的怎么解决

**Comment by ceciliayan** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，邮件已收到。

**Comment by ceciliayan** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，邮件已收到。

**Comment by Yunacn** [comment]
请问导入之后，出现了这个提示要怎么办呢TypeError: Zotero.ZoteroStyle.api.renderCell is not a Function. 我使用的Ethereal Style版本是5.6.3

**Reply by Yunacn** [reply]
.。。。。。重新安装style插件之后就好了

**Reply by YichenPang** [reply]
hhh非常感谢 遇到了同样的问题还以为是我自己瞎捣鼓捣鼓坏了

**Comment by laijiephd** [comment]
zotero 7 报错了，咋办？Template Preview Error: TypeError: Zotero.Items.get(...).filter(...)[0] is undefined

**Comment by caroxx91** [comment]
zotero7，已安装style、better notes、translate 等插件，导入模版笔记点击保存时，一直显示Template [item][item]125 Error: TypeError: Zotero.ZoteroStyle is undefined

**Reply by jyflt** [reply]
我也是这个问题，请问最后如何解决的

**Comment by ceciliayan** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，邮件已收到。

[a comment by an automated account is not shown]

**Comment by ymg-glitch** [comment]
作者大大，什么时候会更新到8呀

**Comment by YichenPang** [comment]
您好，我已收到您的邮件，祝您生活愉快

**Comment by ceciliayan** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，邮件已收到。

**Comment by ceciliayan** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，邮件已收到。

**Comment by YichenPang** [comment]
您好，我已收到您的邮件，祝您生活愉快
